### PR TITLE
components 2

### PR DESCRIPTION
**Pull request by TashaUA**

### CODE REVIEW

File: src/containers/AddEditMovie.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            release_date: '',
+            movie_url: '',
+            genres: [],
+            overview: '',
+            runtime: '',
+            genresOptions: {
+                action: 'Action',
+                adventure: 'Adventure',
+                comedy: 'Comedy'
+            }
```
Comment (lotrien) [inline src/containers/AddEditMovie.jsx]: I don't think `genresOptions` collection belongs to state, it's not something that will be changed, it's more like a constant

File: src/containers/AddEditMovie.jsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    updateUser = (id, data) => {
+        return false;
+    };
+
+    handleInput = (event) => {
+        const target = event.target;
+        const value = target.value;
+        const name = target.name;
+
```
Comment (lotrien) [inline src/containers/AddEditMovie.jsx]: es6 destruction can easily be used here: `const { name, value } = event.target`;

File: src/components/MovieCard.jsx [hunk 3]
```diff
@@ -12,6 +14,8 @@ export default function MovieCard(props) {
                 <span>{release_date.split('-')[0]}</span>
             </p>
             {genres.join(", ")}
+            <AddEditMovie entry={props.entry} />
+            <DeleteMovie id={props.entry.id} />
```
Comment (lotrien) [inline src/components/MovieCard.jsx]: I believe if you open react dev tools components tab -> you'll see that every MovieCard will have an instance of each modal which is not good. 
From my point of view, modals should be placed within MoviesList

File: src/containers/AddEditMovie.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            [name]: value
+        });
+    };
+
+    render() {
+        return (
+            <>
+                <a href="#" className="add-movie-button"
+                   onClick={this.showModal}>{this.state.isEdit ? 'Edit' : '+ Add movie'}</a>
+                <Dialog onClose={this.hideModal} show={this.state.show}
```
Comment (lotrien) [inline src/containers/AddEditMovie.jsx]: modal should be somewhat dumb component and should not handle it's own visibility.
it;s better to do within parent component f.e.`{this.state.show && <Dialog />}`

File: src/containers/AddEditMovie.jsx (hunk 1 above)
Comment (TashaUA) [reply src/containers/AddEditMovie.jsx]: Thanks, fixed

File: src/containers/AddEditMovie.jsx (hunk 2 above)
Comment (TashaUA) [reply src/containers/AddEditMovie.jsx]: also this done

File: src/components/MovieCard.jsx (hunk 3 above)
Comment (TashaUA) [reply src/components/MovieCard.jsx]: refactored

File: src/containers/AddEditMovie.jsx (hunk 4 above)
Comment (TashaUA) [reply src/containers/AddEditMovie.jsx]: done! Thanks for comments

**Pull request merged**